protected/humhub/modules/content/widgets/stream/StreamEntryWidget.php

```diff
@@ -102,15 +102,31 @@ private static function renderLegacyWallEntry(ContentActiveRecord $record, $opti
             $options = [];
         }
 
+        // Ensure that $record->getWallEntryWidget() returns a valid object
+        $wallEntryWidget = $record->getWallEntryWidget();
+        if (!$wallEntryWidget || !$wallEntryWidget->renderOptions) {
+            // Handle the case where $wallEntryWidget or renderOptions is null
+            return;
+        }
+
+        // Ensure that $wallEntryWidget->renderOptions is properly initialized before accessing its properties
+        $jsWidget = null;
```

Comment:
no need to define the `$jsWidget` here, because below the var is defined anyway.